enrichment_logbook_uploader/logbook_upload.py

```diff
@@ -109,6 +110,33 @@ def generate_payload(df, logbook_header_id):
         for date in skipped_dates:
             print(date)
 
+    get_logBook_response = requests.post(_ACTIVITY_ENRICHMENT_URL + "/LogBook/GetLogBook", headers=headers, data="logBookHeaderID="+logbook_header_id)
+
+    emptyCounter = 0
+    dateList = {}
+    date = 0
+    for days in json.loads(get_logBook_response.text)["data"]:
+        if days["id"] == "00000000-0000-0000-0000-000000000000":
+            dateList[date] = days["date"]
```

Comment:
If you change dateList to a regular list, you'll be able to append to it directly
```suggestion
            date_list.append(date)
```